From d3ad013fc1c539c99ee6e37b96a980c27b98cd46 Mon Sep 17 00:00:00 2001
From: Philippe Nunes <philippe.nunes@cortus.com>
Date: Fri, 3 Aug 2018 10:37:40 +0200
Subject: [PATCH 2/8] Build only the D7A module

---
 stack/CMakeLists.txt | 12 +++++-------
 1 file changed, 5 insertions(+), 7 deletions(-)

diff --git a/stack/CMakeLists.txt b/stack/CMakeLists.txt
index 115d083..c2df297 100644
--- a/stack/CMakeLists.txt
+++ b/stack/CMakeLists.txt
@@ -67,7 +67,7 @@ ENDIF()
 EXPORT_GLOBAL_INCLUDE_DIRECTORIES("${CMAKE_CURRENT_BINARY_DIR}/modules")
 
 #Then load the Framework
-ADD_SUBDIRECTORY("framework")
+#ADD_SUBDIRECTORY("framework")
 
 #Load the file with platform-specific build commands
 IF(EXISTS "${PROJECT_SOURCE_DIR}/cmake/platform_${PLATFORM}_macros.cmake")
@@ -76,14 +76,12 @@ ELSE()
     include ("cmake/platform_default_macros.cmake")
 ENDIF()
 
+EXPORT_GLOBAL_INCLUDE_DIRECTORIES(framework/inc)
+EXPORT_GLOBAL_INCLUDE_DIRECTORIES(framework/hal/inc)
+
 SET(MODULE_D7AP ON CACHE BOOL "") # D7AP module enabled by default
 SET(MODULE_D7AP_FS ON CACHE BOOL "")  # D7AP_FS module enabled by default
-SET(MODULE_ALP ON CACHE BOOL "")  # ALP module enabled by default
+#SET(MODULE_ALP ON CACHE BOOL "")  # ALP module enabled by default
 
 #Then load the modules
 ADD_SUBDIRECTORY("modules")
-#And finally the applications
-ADD_SUBDIRECTORY("apps")
-#And tests
-ADD_SUBDIRECTORY("tests")
-
-- 
2.7.4

